packages/dev/core/src/Engines/engineFeatures.ts

```diff
@@ -42,6 +42,9 @@ export interface EngineFeatures {
     /** Indicates that SSAO2 is supported */
     supportSSAO2: boolean;
 
+    /** Indicates that SSR Rendering Pipeline is supported */
+    supportSSR: boolean;

```

Comment:
Should it be based on lower level features instead ? as SSR is not an engine feature I am afraid of the flag multiplication ?